fastlane/Fastfile

```diff
@@ -308,22 +314,22 @@ import "./ScreenshotFastfile"
 
     gym(
       scheme: "WordPress Alpha",
-      workspace: "../WordPress.xcworkspace",
+      workspace: WORKSPACE_PATH,
       export_method: "enterprise",
       clean: true,
-      output_directory: "../build/",
-      derived_data_path: "../derived-data/alpha/",
+      output_directory: BUILD_PRODUCTS_PATH,
+      derived_data_path: DERIVED_DATA_PATH,
       export_team_id: ENV["INT_EXPORT_TEAM_ID"],
-      export_options: { method: "enterprise" })
-
-    sh("mv ../../build/WordPress.ipa \"../../build/WordPress Alpha.ipa\"")
+      export_options: { method: "enterprise" }
```

Comment:
Hmm good point – I might make it so that this just moves them to the `Artifacts` directory that we already have – that way it's consistent across the whole project – if you want the `.ipa` file locally, it'll always be in `Artifacts` (which is what we use for the translation lane)

Reply:
This is adjusted in b15122eac37aa7cb79263daa86d1995c8012112a, which puts the build artifacts for all lanes in the `Artifacts` directory.